src/java/org/apache/cassandra/metrics/StorageMetrics.java

```diff
@@ -45,6 +45,7 @@
     public static final Counter totalHintsInProgress  = Metrics.counter(factory.createMetricName("TotalHintsInProgress"));
     public static final Counter totalHints = Metrics.counter(factory.createMetricName("TotalHints"));
     public static final Counter repairExceptions = Metrics.counter(factory.createMetricName("RepairExceptions"));
+    public static final Counter errorDecommissiong = Metrics.counter(factory.createMetricName("ErrorDecommissioning"));
```

Comment:
@jaydeepkumar1984 Why do we need metrics for knowing if decommission failed? What do you plan to do if you determine that the decommission has failed?